rustls/src/webpki/verify.rs

```diff
@@ -144,68 +142,6 @@ impl<'a> TryFrom<&'a CertificateDer<'a>> for ParsedCertificate<'a> {
     }
 }
 
-/// A `WebPkiSupportedAlgorithms` value that reflects webpki's capabilities when
```

Comment:
Would be nice to move this to its new place in a separate commit.

Reply:
Have done that.